ai.menlo.jan.metainfo.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>ai.menlo.jan</id>
+  <metadata_license>FSFAP</metadata_license>
+  <project_license>AGPL-3.0-only</project_license>
+  <name>Jan</name>
+  <summary>Local AI Assistant that runs 100% offline on your device</summary>
+  
+  <description>
+    <p>
+      Jan is a ChatGPT-alternative that runs 100% offline on your device. Our goal is to make it easy for anyone to download and run LLMs and use AI with full control and privacy.
+    </p>
+    <p>Features:</p>
+    <ul>
+      <li>Model Library with popular LLMs like Llama, Gemma, Mistral, or Qwen</li>
+      <li>Connect to Remote AI APIs like Groq and OpenRouter</li>
+      <li>Local API Server with OpenAI-equivalent API</li>
+      <li>Extensions for customizing Jan</li>
+    </ul>
+  </description>
+  
+  <launchable type="desktop-id">ai.menlo.jan.desktop</launchable>
+  
+  <screenshots>
+    <screenshot type="default">
+      <image>https://raw.githubusercontent.com/janhq/jan/main/demo.gif</image>
```

Comment:
This should never point to the tip of a branch. because it's not a permanent URL.